### ISSUE TITLE
Can't move group block when background colour not selected

### ISSUE THREAD

**Issue by bobbingwide** [WordPress/gutenberg#16718]
**Describe the bug**
When you create a group of blocks and want to move the whole group, it seems that it's not possible to move it until you've chosen a background colour.

**To reproduce**
1. Add a group block with two blocks within it e.g. paragraph and list item.
2. Select the group using Block navigation
3. Try to move the group using the Up or Down arrows
4. Notice that the inner blocks get moved.

**Expected behavior**
The whole group should be moved.

**Workaround**
Repeat the above steps but this time set a background colour to the Group.
Notice that you can now move the Group block.

**Screenshots**
For a video watch this from 10:54to the end
 https://www.youtube.com/watch?v=9AKgSTaqOPc 

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] 75.0.3770.142

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]  iPad

**Additional context**
- Gutenberg 5.9
- Confirmed by Daniel Richards in this thread on Slack https://wordpress.slack.com/archives/C02QB2JS7/p1563865853064200 

- As an aside, I tried to reproduce the problem this morning working on my iPad. It led me to ask the following questions.

1. How do I go about moving blocks into a group? 
2. How do I drag and drop?
3. How do I select multiple blocks?
For an example of selecting multiple blocks using Chrome with a mouse see the previous minute of the video.

**Comment by bobbingwide** [comment]
Here's another video recorded this morning with Gutenberg 6.1.1
https://www.youtube.com/watch?v=LaE_zp4hDdM

**Comment by bobbingwide** [comment]
Re: these aside questions related to my attempts to use an iPad

> * How do I go about moving blocks into a group?
> * How do I drag and drop?
> * How do I select multiple blocks?

Each question led to the next. I couldn't move existing blocks into a group since I didn't know how to drag and drop a block with my iPad. 

I've now discovered that I need to be more patient. 

- Press on the block moving icon ( double ellipsis ) 
- Wait until a little box appears with a green plus. 
Note: You can hardly see it because it's below your finger. 
- Then you can drag the block. 
- And if you're really lucky you'll see the blue line when dropping is possible. 


I couldn't use the _convert to Group_ option since I don't know how to _select multiple blocks_ on my iPad. I believe this is related to #3629. 
When I'm using my iPad, I shouldn't have to jump through hoops to find out how to do something simple like this.  Keyboard Shortcuts offer no help. Maybe there's a way to use the Block Navigation block.

**Comment by bobbingwide** [comment]
> Maybe there's a way to use the Block Navigation block.

Looks like it's going to be one of those days. Now the Block Navigation tool isn't working.

**Comment by mapk** [comment]
Just tested this out myself and can confirm.

![group-move](https://user-images.githubusercontent.com/617986/61732266-11327400-ad32-11e9-8b37-2a29366bee70.gif)

**Comment by karmatosed** [comment]
I am not sure why this needs a design, shouldn't the block just be able to move no matter what state? I'd love to find out more why this needs a new design and what I might be missing @mapk.

**Comment by mapk** [comment]
@karmatosed, the reason I added "Needs Design" because this was an addition to the series of block mover issues getting recorded lately. I felt another design implementation would be necessary at this point.

However, after testing this again, I am no longer experiencing this issue. Looks like this has been resolved. Yay!! 🎉 

![group](https://user-images.githubusercontent.com/617986/64301842-93b77180-cf35-11e9-81f5-2687287df4ec.gif)


I'm closing this issue for that reason.
